r2dec.js

```diff
@@ -29,6 +29,10 @@ const Json64 = require('./decompile/json64.js');
 const r2pipe = require('r2pipe');
 const util = require('util');
 
+if (!r2pipe.jsonParse) {
```

Comment:
hmm, maybe it's better to use `r2pipe.hasOwnProperty('jsonParse')`
for the version number, i was going to update the `package.json`